### DISCUSSION TITLE
i am a novice. can this project opened using arduino ide .

### DISCUSSION THREAD

**Discussion by AbhishekAKA47** [sekigon-gonnoc/Pico-PIO-USB#74] [Q&A] [answered]
specifically the capture hid report . on my computer when i go to examples pico pio usb i can only find the device info option why cant i see capture hid report in that and how do i get it . i have installed pico visual studio and am trying to install the tinyusb here but dont know how any help ? and where else should i ask for help regarding how to compile this capture hid repost?

**Comment by sfranzyshen** [comment, marked as answer]
try using this project ... https://github.com/adafruit/Adafruit_TinyUSB_Arduino

**Reply by AbhishekAKA47** [reply]
I want to log all the mouse movements and play them back after a certain delay to fool my friend into thinking that is why I am so bad at the game. this is a video to show what I have set up I am on windows 10.

https://user-images.githubusercontent.com/127853293/229354232-e2911bf4-7aa6-4354-9385-4423d7a73bd6.mp4

reply in video if you can so i know what its supposed to look like at least.

**Reply by AbhishekAKA47** [reply]
> try using this project ... https://github.com/adafruit/Adafruit_TinyUSB_Arduino

can you atleast give a screenshot for the examples please?

**Reply by touchgadget** [reply]
@AbhishekAKA47 Lots of examples are installed in the IDE when the Adafruit library is installed. You can also browse the same examples using the following link.

https://github.com/adafruit/Adafruit_TinyUSB_Arduino/tree/master/examples

**Reply by AbhishekAKA47** [reply]
cool
